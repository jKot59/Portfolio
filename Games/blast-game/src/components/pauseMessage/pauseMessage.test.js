import React from 'react';
import PauseMessage from './pauseMessage';
import renderer from 'react-test-renderer';

describe('Testing PauseMessage', () => {
    it('Header have render correctly', () => {
        const pauseMessage = renderer.create(<PauseMessage/>).toJSON()
        expect(pauseMessage).toMatchSnapshot();
    })
})