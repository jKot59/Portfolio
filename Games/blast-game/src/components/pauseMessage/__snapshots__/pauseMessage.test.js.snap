// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing PauseMessage Header have render correctly 1`] = `
<div
  className="sc-bdnxRM fQhPCj"
>
  <h3>
    PAUSED
  </h3>
  <span>
    Ни на что не влияет. Для демонтрации
  </span>
</div>
`;
